docs/assets/firehose-grafana-dashboard-updated.json

```diff
@@ -0,0 +1,10762 @@
+{
```

Comment:
Why are you adding this file entirely exactly. The old Dashboards should be updated, right?

Reply:
fixed